src/helm/benchmark/run_specs.py

```diff
@@ -2516,7 +2516,13 @@ def construct_run_specs(spec: ObjectSpec) -> List[RunSpec]:
         ]
 
     def alter_run_spec(run_spec: RunSpec) -> RunSpec:
-        model = get_model(run_spec.adapter_spec.model)
+        try:
+            model = get_model(run_spec.adapter_spec.model)
+        except ValueError:
+            # Models registered from configs cannot have expanders applied to them,
```

Comment:
ValueError means that the model has not been loaded yet?  I was a bit confused by the comment at first, maybe connect the dots a bit more.

Reply:
It means the model has not been registered yet. I'll add more docs.